spec/requests/content_blocks_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'ContentBlocks', type: :request do
+  let(:user) { create(:user) }
+  let(:content_block) { create(:content_block) }
+
+  before do
+    sign_in user, groups: ['sdr:administrator-role']
+  end
+
+  describe 'GET /content_blocks' do
+    before do
+      create(:content_block)
+      create(:content_block)
+    end
+
+    it 'displays the content blocks' do
+      get content_blocks_path
+      expect(response.body).to include 'Message alerts'
+    end
```

Comment:
might be nice to also test that secondary alerts display below primary alerts, with their specific header text

Reply:
we never test page layout in a request test. In fact it would be impossible to do so because CSS is not loaded.

Reply:
ah, good point